### PR TITLE
Added option seed support for SotV and SotS characters

### PR DESCRIPTION
**Pull request by Phreelosu**
you can take a look at their codes, its a bunch of copy paste from the previously written ones, but they all work really well ingame
and I'd say Railgunner benefits the most out of option seeds
Void Fiend does too but not that crazy of a buff

Seeker was a tiny bit difficult since Seeker's primary fires a different projectile every 3rd shot, so I just made a quick workaround so the orbs fire the finisher projectile every 3rd skill use

Chef is an annihilator, especially how "Yes Chef" skill's enhanced primary is also directly affected, so Chef practically launches 20+ butcher knives per enhanced primary LOL

False Son benefits the least from option seeds, I'd say to the same extent as Acrid or even less, False Son is extremely close ranged and has mainly melee attacks, so Lunar Spikes are the only affected skills and its not that great of a buff, will not save runs if anything

oh and I also disabled the multiplayer testing cuz it was accidentally left in, oops

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,3 @@
-![Version](https://img.shields.io/badge/Version-3.5.4-orange)
-[![GitHub issues](https://img.shields.io/github/issues/cheeeeeeeeeen/RoR2-ChensGradiusMod)](https://github.com/cheeeeeeeeeen/RoR2-ChensGradiusMod/issues)
-[![GitHub pull requests](https://img.shields.io/github/issues-pr/cheeeeeeeeeen/RoR2-ChensGradiusMod)](https://github.com/cheeeeeeeeeen/RoR2-ChensGradiusMod/pulls)
-[![Support Chen](https://img.shields.io/badge/Support-Chen-ff69b4)](https://ko-fi.com/cheeeeeeeeeen)
-![Maintenance Status](https://img.shields.io/badge/Maintenance-Inactive-orange)
-
```
Comment (cheeeeeeeeeen) [inline README.md:6]: You can actually just change the version number here and the badges will still work. Probably just remove the Support Chen and PR ones.

Comment (Phreelosu) [reply README.md:6]: latest commit should resolve this, also added Inspect mode support that was introduced in the new DLC

Comment (cheeeeeeeeeen) [reply README.md:6]: What's inspect mode again?

Comment (cheeeeeeeeeen) [reply README.md:6]: Ah nvm. A simple search did it. Alright.

**Pull request merged**